### PR TITLE
Add upgrade OCP version CLI options

### PR DESCRIPTION
**Pull request by clacroix12**
Jira: [OCSQE-232](https://projects.engineering.redhat.com/browse/OCSQE-232)

Signed-off-by: Coady LaCroix <clacroix@redhat.com>

### CODE REVIEW

File: ocs_ci/framework/pytest_customization/ocscilib.py [hunk 1]
```diff
@@ -142,6 +142,11 @@ def pytest_addoption(parser):
         dest='upgrade_ocs_version',
         help="ocs version to upgrade (e.g. 4.3)"
     )
+    parser.addoption(
+        '--upgrade-ocp-version',
+        dest='upgrade_ocp_version',
+        help='ocp version to upgrade to (e.g. 4.5)'
```
Comment (petr-balogh) [inline ocs_ci/framework/pytest_customization/ocscilib.py:148]: The info should look like more similar to --ocp-version parameter where we accept -ga versions as well.
So I would add examples like we have in there:
```
        4.2: for nightly 4.2 OCP build
        4.2-ga: for latest GAed 4.2 OCP build
```

Comment (clacroix12) [reply ocs_ci/framework/pytest_customization/ocscilib.py]: Done

**Review by petr-balogh — APPROVED** [review]

**Review by shylesh — APPROVED** [review]
+1

**Review by AviadP — APPROVED** [review]
LGTM

**Pull request merged**